app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,72 @@
+module Api
+  module V1
+
+    class TasksController < Api::ApplicationController
+      before_action :set_project
+      before_action :set_task, only: %i[destroy update]
+
+      def index
+        @tasks = @project.tasks
+                         .order(params[:sort]).page(params[:page]).per(3)
+
+        serializable_tasks = ActiveModelSerializers::SerializableResource.new(
+        @tasks, each_serializer: TaskSerializer
+        )
+        render json: { tasks: serializable_tasks }
+      end
+
+      def create
```

Comment:
Please use here serializers too (for update too)

```
def update
        @task = update_task.task
        serializable_task = ActiveModelSerializers::SerializableResource.new(
          @task, serializer: TaskSerializer
        )
        if update_task.success?
          render json: { task: serializable_task, message: "Task Updated" }
        else
          render json: { task: {}, errors: @task.errors }
        end
      end
 ```

Reply:
Did it. Tho why exactly did I need to add the serializable_task here as the functionality ain't affected without it?

Reply:
For practice, because you use seriallizer only for index action, but usually it use for all actions